### ISSUE TITLE
why you did not used lstm

### ISSUE THREAD

**Issue by Sandy4321** [sjvasquez/instacart-basket-prediction#8]
why you did not used lstm
in
Product RNN mixture model (code): an RNN similar to the first model, but instead trained to maximize the likelihood of a bernoulli mixture model

**Comment by metatron1973** [comment]
Simple Instacart is fraudulent at all levels remove the corruption ![image](https://user-images.githubusercontent.com/80199202/177381684-63ec779c-99e1-4ad3-a16b-16b04d394a41.png)

**Comment by metatron1973** [comment]
Still major fraud inside system

**Comment by metatron1973** [comment]
No![image](https://user-images.githubusercontent.ncom/80199202/209588685-81126404-0fab-46f7-962e-113f42d487c7.png) none of this matters when they are stealing driver tips current forensic audit in full swing

**Comment by metatron1973** [comment]
![image](https://user-images.githubusercontent.com/80199202/209588779-a4542e57-7b9d-40fd-8a93-54a54d72f057.png) is it ok for a company to steal wages during holidays ?
